ee/ucp/admin/install/install-on-azure.md

```diff
@@ -4,81 +4,79 @@ description: Learn how to install Docker Universal Control Plane in a Microsoft
+
+- _An automated mechanism provided by UCP which allows for IP pool configuration and maintenance
+  for standalone Azure virtual machines._ This service runs within the
+  `calico-node` daemonset and provisions 128 IP addresses for each
+  node by default. For information on customizing this value, see [Adjusting the IP count value](#adjusting-the-ip-count-value).
+- _Manual provision of additional IP address for each Azure virtual machine._ This
+  could be done through the Azure Portal, the Azure CLI `$ az network nic ip-config create`,
+  or an ARM template. You can find an example of an ARM template
+  [here](#manually-provision-ip-address-as-part-of-an-azure-virtual-machine-scale-set).
 
 ## Azure Prerequisites 
 
-You must meet these infrastructure prerequisites in order 
-to successfully deploy Docker UCP on Azure
+You must meet the following infrastructure prerequisites in order 
+to successfully deploy Docker UCP on Azure:
 
-- All UCP Nodes (Managers and Workers) need to be deployed into the same 
-Azure Resource Group. The Azure Networking (Vnets, Subnets, Security Groups) 
-components could be deployed in a second Azure Resource Group.
-- The Azure Vnet and Subnet must be appropriately sized for your
-  environment, and addresses from this pool are consumed by Kubernetes Pods. For more information, see
-  [Considerations for IPAM
+- All UCP Nodes (Managers and Workers) need to be deployed into the same Azure
+  Resource Group. The Azure Networking components (Virtual Network, Subnets,
+  Security Groups) could be deployed in a second Azure Resource Group.
+- The Azure Virtual Network and Subnet must be appropriately sized for your
+  environment, as addresses from this pool will be consumed by Kubernetes Pods.
+  For more information, see [Considerations for IPAM
   Configuration](#considerations-for-ipam-configuration).
-- All UCP Nodes (Managers and Workers) need to be attached to the same 
-Azure Subnet.
-- All UCP (Managers and Workers) need to be tagged in Azure with the 
-`Orchestrator` tag. Note the value for this tag is the Kubernetes version number
-in the format `Orchestrator=Kubernetes:x.y.z`. This value may change in each 
-UCP release. To find the relevant version please see the UCP 
-[Release Notes](../../release-notes). For example for UCP 3.1.0 the tag 
-would be `Orchestrator=Kubernetes:1.11.2`. 
-- The Azure Virtual Machine Object Name needs to match the Azure Virtual Machine 
-Computer Name and the Node Operating System's Hostname. Note this applies to the 
-FQDN of the host including domain names. 
-- An Azure Service Principal with `Contributor` access to the Azure Resource 
-Group hosting the UCP Nodes. Note, if using a separate networking Resource 
-Group the same Service Principal will need `Network Contributor` access to this 
-Resource Group.
+- All UCP worker and manager nodes need to be attached to the same Azure
+  Subnet.
+- All UCP workers and managers need to have the `Orchestrator` tag in Azure with
```

Comment:
I don't recall who exactly consumes the `orchestrator` label from Azure - core kubernetes, UCP controllers or Calico? If a node were to be switched from UCP to Swarm or vice versa in UCP UI, does the label need to be added/removed out-of-band as well? This needs to be clarified by someone from UCP. @wsong any thoughts?

Reply:
I'm actually not familiar with this Orchestrator tag; UCP and Swarm certainly never consume it. Is this for the Kube cloud provider code, or does this tag somehow get translated to a Swarmkit node label?

Reply:
Thanks Wayne. @ollypom PR: https://github.com/docker/docker.github.io/pull/7353 introduced the Orchestrator label as a pre-req for Azure nodes. Do you happen to recall the context around this - which component is the consumer of these labels?

Reply:
@ddebroy The original pre-req came from @jtestard's notes here https://github.com/docker/orca/issues/13772 for Azure Disk. I know that @joeabbey looked at it more detail and didn't think it was required https://github.com/docker/orca/issues/16019. As that issue remained opened though, I've never removed it from the Public docs as precaution.

Reply:
Thanks for the context, @ollypom . Can we remove this please?

Reply:
@ddebroy Thanks, I've removed this prereq.